test_data/proxy/src/com/intellij/memory/agent/MemoryAgentException.java

```diff
@@ -0,0 +1,11 @@
+package com.intellij.memory.agent;
+
+public class MemoryAgentException extends Exception {
```

Comment:
Perhaps worth adding more detailed exceptions like `MemoryAgentInitializationException` and `MemoryAgentExecutionException`